spec/controllers/works/miscellaneous_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe WorksController do
+  include LoginMacros
+  include RedirectExpectationHelper
+
+  let(:user) { create(:user) }
+  let!(:work) { create(:work, authors: [user.default_pseud]) }
```

Comment:
Does this definitely need to be `let!`? (I only ask because I know `let` works with the regular `edit` action, so I thought it might here as well.)

Reply:
Yeah, should be fine without !, fixed